options_test.go

```diff
@@ -1,68 +1,82 @@
 package lightstep
```

Comment:
Ya, the exact reason I did that was I noticed the existing tests access a bunch of private fields. I can change the package, but I also would have to delete most of the existing tests